src/components/DatePicker.tsx

```diff
@@ -1,8 +1,26 @@
+import InteractionRing from "@leafygreen-ui/interaction-ring";
+import { uiColors } from "@leafygreen-ui/palette";
 import generatePicker from "antd/lib/date-picker/generatePicker";
 import dateFnsGenerateConfig from "rc-picker/lib/generate/dateFns";
+import { size } from "constants/tokens";
 
 import "antd/lib/date-picker/style/css";
 
+const { gray } = uiColors;
+
 const DatePicker = generatePicker<Date>(dateFnsGenerateConfig);
 
-export default DatePicker;
+export default (props) => {
+  const { disabled } = props;
+  return (
+    <InteractionRing disabled={disabled}>
+      <DatePicker {...props} style={leafygreenInputStyle} />
+    </InteractionRing>
+  );
+};
+
+const leafygreenInputStyle = {
+  border: `1px solid ${gray.base}`,
+  borderRadius: size.xxs,
+  transition: "border-color 150ms ease-in-out",
+};
```

Comment:
I decided to use [antd's `style` prop](https://ant.design/components/date-picker/#Common-API) directly since it specifically targets the `input` element. This seems more future-proof than targeting a node that could be subject to future changes to the component's internals.